examples/list_displays_async.rs

```diff
@@ -25,12 +27,9 @@ async fn show_brightness(dev: &BrightnessDevice) -> Result<(), brightness::Error
 
 #[cfg(windows)]
 async fn show_platform_specific_info(dev: &BrightnessDevice) -> Result<(), brightness::Error> {
-    use brightness::BrightnessExt;
-    println!("\tDevice description = {}", dev.device_description().await?);
-    println!(
-        "\tDevice registry key = {}",
-        dev.device_registry_key().await?
-    );
+    use brightness::windows::BrightnessExt;
```

Comment:
the extension is now under the `windows` module for consistency with the standard library, e.g. `std::os::unix::fs::PermissionsExt`